zcutil/fetch-params.sh

```diff
@@ -225,7 +234,7 @@ EOF
     fetch_params "$SAPLING_SPROUT_GROTH16_NAME" "$PARAMS_DIR/$SAPLING_SPROUT_GROTH16_NAME" "b685d700c60328498fbde589c8c7c484c722b788b265b72af448a5bf0ee55b50"
 }
 
-if [ "x${1:-}" = 'x--testnet' ]
+if [ "${1:-}" = '--testnet' ]
```

Comment:
Yes the rationale for the "x" thing is wrong; there's never any ambiguity for `test "$a" = 'b'` or `[ "$a" = 'b' ]`: https://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes/174288#comment54562572_180630